config/locales/pt.yml

```diff
@@ -25,6 +25,7 @@ pt:
         email: 'E-mail'
         birthday: 'Data de Nascimento'
         country: 'País'
+        citizen_of: 'Cidadão de'
```

Comment:
Yes, it's acceptable.

On Sat, Jul 25, 2015 at 4:10 PM, Florian Weingarten <
notifications@github.com> wrote:

> In config/locales/pt.yml
> <https://github.com/fw42/cubecomp/pull/194#discussion_r35483777>:
>
> > @@ -25,6 +25,7 @@ pt:
> >          email: 'E-mail'
> >          birthday: 'Data de Nascimento'
> >          country: 'País'
> > +        citizen_of: 'Cidadão de'
>
> @cubizh <https://github.com/cubizh> is this correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fw42/cubecomp/pull/194/files#r35483777>.
>